test/entropic/sinkhorn_epsscaling.jl

```diff
@@ -0,0 +1,87 @@
+using OptimalTransport
+
+using Distances
+using ForwardDiff
+using LogExpFunctions
+using PythonOT: PythonOT
+
+using LinearAlgebra
+using Random
+using Test
+
+const POT = PythonOT
+
+Random.seed!(100)
+
+@testset "sinkhorn_epsscaling.jl" begin
+    # size of source and target
+    M = 250
+    N = 200
+
+    # create two random histograms
+    μ = normalize!(rand(M), 1)
+    ν = normalize!(rand(N), 1)
+
+    # create random cost matrix
+    C = pairwise(SqEuclidean(), rand(1, M), rand(1, N); dims=2)
+
+    # regularization parameter
+    ε = 0.01
+
+    @testset "example" begin
+        # compute optimal transport plan and cost with POT
+        γ_pot = POT.sinkhorn(μ, ν, C, ε; method="sinkhorn_stabilized", numItermax=5_000)
+        c_pot = POT.sinkhorn2(μ, ν, C, ε; method="sinkhorn_stabilized", numItermax=5_000)[1]
+
+        for alg in (SinkhornGibbs(), SinkhornStabilized())
+            # compute optimal transport plan and cost
+            γ = sinkhorn(μ, ν, C, ε, SinkhornEpsilonScaling(alg); maxiter=5_000)
+            c = sinkhorn2(μ, ν, C, ε, SinkhornEpsilonScaling(alg); maxiter=5_000)
```

Comment:
I think we should use `cost` and leave `c` to be the OT cost function, while `C` is the cost matrix.

Reply:
After reading the rest of the code for the tests, I see that this would require a huge amount of refactoring in the tests, so I think it's better to just leave it be.

Reply:
I think more consistent names are a good idea but I assume it might be better to do this in a separate PR since it might also affect other unrelated parts.